src/plugins/dam/components/image-asset-picker-field/index.jsx

```diff
@@ -78,12 +83,41 @@ class ImageAssetPickerField extends React.Component {
       label,
       meta: { error },
       node,
+      getNode,
     } = this.props;
 
+    const selected = getNode(value);
```

Comment:
nit: what do you think about naming the variable to something a little more specific like `image` (or `imageNode` or something) instead of `selected`.